impeller/entity/contents/tessellation_cache.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/entity/contents/tessellation_cache.h"
+
+#define IMPELLER_ENABLE_TESSELLATION_CACHE 1
+
+namespace impeller {
+
+Path::Polyline TessellationCache::GetOrCreatePolyline(
+    const impeller::Path& path,
+    Scalar scale) {
```

Comment:
If you take the exact scale as a cache key then animated transformations will thrash your cache each frame despite the vertices being almost unchanged.